README.md

```diff
@@ -9,7 +9,12 @@ A command-line tool which simplifies the task of updating your Flutter app's lau
 
 ## :sparkles: What's New
 
-#### 0.8.1 (2nd Oct 2020)
+#### Version 0.9.0 (10th Nov 2020)
```

Comment:
Make sure to update the release date (and possibly select a different version, if another is more appropriate).

Reply:
I always update the release date before release 😃